lib/google/widgets/translate_button/translate_button_float.dart

```diff
@@ -51,14 +51,15 @@ class TranslateButtonFloat extends StatelessWidget {
                           }
                         }
                       : null,
-              child: Text(AppLocalizations.of(context)!.translate,
-                  style: TextStyle(
-                      fontSize: 18,
-                      color: theme == Brightness.dark
-                          ? null
-                          : googleTranslationInputController.text == ''
-                              ? lightThemeGreyColor
-                              : Colors.white)),
+              child: Text(
```

Comment:
Everything is fine, only for text strings you adopted this rule earlier
```
              child: Text(
                AppLocalizations.of(context)!.translate,
```
and I corrected it so that it is the same in the code. It can also make it easier for translators to find the code to see what function it refers to.

Reply:
And the remaining lines were adjusted to the displacement of the previous one 😁.

Reply:
This is the dart auto formatter on VS-Code:

https://user-images.githubusercontent.com/40805353/134865265-3ad973e7-6408-4241-9298-dee659050df9.mp4

As you can see, you should add a comma at the end of the list to trigger it.